### PR TITLE
Handling if JsonResponseHandler receives a blank body 

### PR DESCRIPTION
**Pull request by dgulinobw**
Returning response object instead.

### CODE REVIEW

File: apiclient/response_handlers.py [hunk 1]
```diff
@@ -33,7 +33,10 @@ class JsonResponseHandler(BaseResponseHandler):
     @staticmethod
     def get_request_data(response: Response) -> JsonType:
         try:
-            response_json = response.json()
+            if response.text == "":
+                response_json = response
```
Comment (MikeWooster) [inline apiclient/response_handlers.py:37]: Shouldn't this function return a blank string in this case?

Comment (dgulinobw) [reply apiclient/response_handlers.py:37]: I had originally had it returning a blank string, but then switched it to the behavior of RequestResponseHandler, which is to return the response.  Either way works, I just wasn't sure what to be consistent with.

Comment (MikeWooster) [reply apiclient/response_handlers.py:37]: I think in this case, if there is no body, then returning `None` would be appropriate, explicitly indicating that there is no content.

**Comment by MikeWooster** [comment]
fixed this in https://github.com/MikeWooster/api-client/pull/38

**Pull request closed without merging**